api/peripheral_api/routes/LedSign.js

```diff
@@ -23,14 +21,7 @@ router.get('/healthCheck', async (req, res) => {
   if (process.env.NODE_ENV !== 'production') {
     return res.sendStatus(OK);
   }
-  await axios
-    .get('http://host.docker.internal:11000/api/health-check')
-    .then(() => {
-      return res.sendStatus(OK);
-    })
-    .catch((err) => {
-      return res.sendStatus(NOT_FOUND);
-    });
+  await updateSign(...req.body);
```

Comment:
switch these, we are calling update sign in health check and health check in update sign